### PR TITLE
PEP8ification

### PR DESCRIPTION
**Pull request by Neraste**
Beware the mighty PEP8!

This PR applies the PEP8 to the whole project, as well as it provides tests scripts for CI.

Over minor improvements:
* The base `FontLoader` class is now abstract;
* Tests have been moved to the separate folder `dakara_player_vlc/tests`;
* Use absolute imports;
* Limit the logged response of the server in case of error to 100 characters.

### CODE REVIEW

File: dakara_player_vlc/dakara_server.py [hunk 1]
```diff
@@ -220,21 +218,28 @@ def send_status_get_commands(self, playing_id, timing=0, paused=False):
...
 
+def display_message(message, limit=100):
+    """Display the 100 first characters of a message
+    """
+    if len(message) <= limit:
+        return message
+
+    return message[:limit].strip() + "..."
+
+
```
Comment (Nadeflore) [inline dakara_player_vlc/dakara_server.py]: So, now the log messages are limited in length
what is the motivation behind this change?

Comment (Neraste) [reply dakara_player_vlc/dakara_server.py]: My bad, I forgot to list this feature.

Sometimes in development when there are errors when communicating with the server, the response of the server is an entire HTML page, which floods the logs.

Comment (Neraste) [reply dakara_player_vlc/dakara_server.py]: Maybe I should shrink in case of HTML responde only ?
